.github/workflows/lint.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Set up node ${{ matrix.node-versions }}
         uses: actions/setup-node@v1
         with:
-          node-versions: ${{ matrix.node-versions }}
+          node-version: ${{ matrix.node-versions }}
```

Comment:
Yeah, we're also using `${{ matrix.node-version }}`

Reply:
Should I rename those as well? Because the one with s is the one from strategy and I didn't want to rename everything, but I could do so, if you wish.

Reply:
I dunno.
We all copy the one from apptutorial, this is the base model

Reply:
Fixed there already: https://github.com/nextcloud/app-tutorial/commit/410497401e07e908bc6318e3f9e2b3480e7aa294